components/resources/wallet_strings.grdp

```diff
@@ -99,7 +99,7 @@
   <message name="IDS_BRAVE_WALLET_REVIEW_ORDER" desc="Swap tab Review order button">Review order</message>
   <message name="IDS_BRAVE_WALLET_REVIEW_SEND" desc="Send tab Review order button">Review send</message>
   <message name="IDS_BRAVE_WALLET_NO_AVAILIBLE_TOKENS" desc="Send tab placeholder for empty token list.">No available tokens</message>
-  <message name="IDS_BRAVE_WALLET_SEARCH_TOKENS" desc="Send tab search for tokens.">Search token by name</message>
+  <message name="IDS_BRAVE_WALLET_SEARCH_TOKENS" desc="Send tab search for tokens.">Search name or paste address</message>
```

Comment:
Suggestion: `Search by name...`

Reply:
Good suggestion! This has been updated.

![Screenshot 33](https://github.com/brave/brave-core/assets/40611140/249616ad-939f-4408-86fc-a731643cb47f)